tools/README.md

```diff
@@ -0,0 +1,28 @@
+###Install
+```
+npm install -g yo@latest
+npm i
+npm link
+```
+
+###Login and Run and Clean up
+```
+oc login https://console.pathfinder.gov.bc.ca:8443 --token=RU4grkYyju_xTt78vROEGRNT6vkhps80qnx_o2RcSkY
```

Comment:
Why is there a token here? Granted should be expired already...

Reply:
It's not a real token, it's there simply as an indicator that you must go get your token before using the generators.

as far as the secrets, this is the exact same create as we have been using in our projects.  i think we should move to using templates for configmaps and secrets, then we can mine the parameters and we can set a lot more useful metadata for those objects too.